src/target/stm32f1.c

```diff
@@ -97,7 +97,9 @@ static int stm32f1_flash_write(struct target_flash *f,
 
 #define FLASHSIZE     0x1FFFF7E0
 #define FLASHSIZE_F0  0x1FFFF7CC
-
+//
+#include "stm32f1_ch32.c"
```

Comment:
Rather than doing this, we'd rather see a new target in the build system even at the expense of some duplicated code.